service/worker/failovermanager/workflow.go

```diff
@@ -48,6 +48,7 @@ const (
 	WorkflowTypeName = "cadence-sys-failoverManager-workflow"
 	// WorkflowID will be reused to ensure only one workflow running
 	WorkflowID             = "cadence-failover-manager"
+	DrillWorkflowID        = WorkflowID + "-drill"
```

Comment:
I thought if we have only one workflow ID we can ensure there's only one failover workflow running at a time?

Reply:
Yes. In a managed failover, it checks if this drill workflow is running. With this, we don't need to terminate the scheduled failover wf. But I am open to use one wf ID